src/seedu/addressbook/AddressBook.java

```diff
@@ -771,10 +771,6 @@ private static void savePersonsToFile(ArrayList<String[]> persons, String filePa
      *
      * @param person to add
```

Comment:
errr,, you seem to have removed `addPersonToAddressBook()` but your commit message says "Remove unused deletePersonFromAddressBook() method"